File: problems/1252/solution.go
package main
func oddCells(n int, m int, indices [][]int) int {
    grid, ans := make([][]int, n), 0
    
    for i, _ := range grid {
        grid[i] = make([]int, m)
    }
    
    for _, ind := range indices {
        ri, ci := ind[0], ind[1]
        
        for i := 0; i < m; i++ {
            grid[ri][i]++
        }
        
        for i := 0; i < n; i++ {
            grid[i][ci]++
        }
    }
    
    for i := 0; i < n; i++ {
        for j := 0; j < m; j++ {
            if grid[i][j] % 2 != 0 {
                ans++
            }
        }
    }
    
    return ans
}

File: problems/1252/solution.py
class Solution:
    def oddCells(self, n: int, m: int, indices: List[List[int]]) -> int:
        r = [0] * n
        c = [0] * m
        
        for k in indices:
            r[k[0]] += 1
            c[k[1]] += 1
        
        count = 0
        
        for i in r:
            for j in c:
                if (i + j) % 2 == 1:
                    count += 1
        
        return count